### ISSUE TITLE
Module 3 Suggestions for Improvements

### ISSUE THREAD

**Issue by yuliaUU** [UBC-MDS/ds-toolbox#113]
I could not find where the transcripts are stored so I can edit them

1.  Slides:

- Wildcards: would be nice to explain what `touch` command  does in a transcript, it is explained in the video
- Need to tell how to exit from `man` command:  type `q` and enter: may be it will be a better last question on the last quiz?

2. Videos:

- 1 slide 4: text like command-line or cli or terminal should appear on the video slides. terminal emulator not defined
- 1 slide 5: would be great to show a small video how to search for a terminals on Mac and Windows. Because so many new names are thrown at the students, that it is hard to follow
- 3 slide 2: it would be very helpful to provide students with zip folder and ask them to download it on their computer  in the location of the `pwd` output so they can follow along. Cause saying that you can "create this file structure using graphical file browser" most likely mean nothing for them 
- 3 slide 2 : also would be nice to show the repo organization using the :visual: interface on mac. Not sure how many students see the repo structures like that using lines and indentations
- 3 slide 4 mnemonics should appear on a video slide
- 3 slide 7 terms such as current or parent directory needs to appear on the video
- 3 slide 9: `rm` wont work with non-empty folders
- - 5 slide 7: would be nice to show an image from the assignment 1 where we set up aliases to references old assignment  where they are asked to set an alias for `mv`, `copy`, `mv` to always ask when you want  to overwrite the files. 

3. General comments for the module:

- I think any commands that are mentioned needs to appear on the video as a text: cause it is very hard to hear "we can write dash F   dash a as dash Fa"
- whenever you navigate to a certain place: it would be great to have a image of a folder structure image: so students can see what are subfolders what are the parent directory and etc

**Comment by joelostblom** [comment]
The transcript are stored in the slides files, under the `Notes` tag in each slides files. E.g. this is the transcript for module 3, slide 1:

https://github.com/UBC-MDS/ds-toolbox/blob/fbbc88f7bfd3accabda917ea4f5c56c9679dbb89/chapters/en/slides/module3/module3_01.md?plain=1#L17-L19

I like both the slides suggestion you made, so feel free to make a PR for those! Possibly we could also add the zip folder as a link in the transcript (but this is not a priority if it is time-consuming)? I also agree with you with many of the changes to the video and we should take those into consideration if we get resources to remake the videos (e.g. showing the commands or at least highlighting them would be great since it is difficult to listen when commands are read out loud)

**Comment by yuliaUU** [comment]
Just a note:  the transcripts have quite extensive description on how to use `man` command, so no changes needed there. I guess for the future would be great to add those descriptions in the video too
![image](https://user-images.githubusercontent.com/54612866/223175711-e3d6dfd8-cc64-433a-bc62-3fd0abb8655a.png)
